packages/web-components/src/field/field.ts

```diff
@@ -79,41 +102,62 @@ export class Field extends FASTElement {
    *
    * @public
    */
+  @observable
   public input!: SlottableInput;
 
+  /**
+   * Updates the field's states and label properties when the assigned input changes.
+   *
+   * @param prev - the previous input
+   * @param next - the current input
+   */
+  public inputChanged(prev: SlottableInput | undefined, next: SlottableInput | undefined) {
+    if (next) {
+      this.setStates();
+      this.setLabelProperties();
+    }
+  }
+
   /**
    * Calls the `setStates` method when a `change` event is emitted from the slotted input.
    *
    * @param e - the event object
    * @internal
    */
-  public changeHandler(e: Event): void {
+  public changeHandler(e: Event): boolean | void {
     this.setStates();
+    this.setValidationStates();
+
+    return true;
   }
 
   /**
    * Redirects `click` events to the slotted input.
    *
+   * @param e - the event object
    * @internal
    */
   public clickHandler(e: MouseEvent): boolean | void {
-    if (this.isSameNode(e.target as Node | null)) {
-      this.input.focus();
+    if (this === e.target) {
       this.input.click();
-      return;
     }
 
     return true;
   }
 
+  constructor() {
+    super();
+    this.elementInternals.role = 'presentation';
```

Comment:
I'm curious why you added this, I would expect it to remove semantic meaning from all of its children. Is the idea here that because we explicitly set roles on all children it's okay? What if someone tried to sneak a semantic element in (e.g... `<em>`?) what would we want to happen?

Reply:
This should be fine - https://www.w3.org/WAI/ARIA/apg/practices/hiding-semantics/

> Common Uses of Role presentation
> 3. Eliminating semantics of intervening orphan elements in the structure of a composite widget, such as a tablist, menu, or tree [...]

> Effects of Role presentation
> 3. The roles, states, and properties of each descendant element remain visible to assistive technologies unless the descendant requires the context of the presentational element. For example:
> * If presentation is applied to a ul or ol element, each child li element inherits the presentation role because ARIA requires the listitem elements to have the parent list element. So, the li elements are not exposed to assistive technologies, but elements contained inside of those li elements, including nested lists, are visible to assistive technologies.